beacon/engine/types.go

```diff
@@ -20,12 +20,19 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/holiman/uint256"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/trie"
 )
 
+var (
+	OldBlockMillisecondsInterval uint64 = 1000
+	NewBlockMillisecondsInterval uint64 = 500
```

Comment:
It is best to add comments to the hard coded  which may change in the future. It is better to add a config to set this variable later to make to changable.